sqlglot/dialects/exasol.py

```diff
@@ -564,3 +566,205 @@ def rank_sql(self, expression: exp.Rank) -> str:
             if expression.args.get("expressions"):
                 self.unsupported("Exasol does not support arguments in RANK")
             return self.func("RANK")
+
+        def table_sql(self, expression: exp.Table, sep: str = " AS ") -> str:
+            """
+            If a table has PIVOTs attached, let the Pivot render a derived table .
+            """
+            pivots = expression.args.get("pivots") or []
+            if not pivots:
+                return super().table_sql(expression)
+
+            if len(pivots) > 1:
+                self.unsupported("Multiple PIVOT clauses are not supported by Exasol")
+                return super().table_sql(expression)
+            pivot = pivots[0]
+            return self.sql(pivot)
```

Comment:
Why are we generating the pivot instead of the table here? That feels wrong. Shouldn't the whole pivot transpilation happen with a `preprocess` call or something instead of this?